.travis.yml

```diff
@@ -69,14 +72,10 @@ branches:
 
 addons:
   apt:
-    sources:
-      # Need msgfmt 0.19.8 to be able to generate .desktop files
-      - sourceline: 'ppa:ricotz/toolchain'
     packages:
       - autoconf
       - clang
       - lcov
-      - gettext
       - libperl-dev
       - python-dev
       - python3-dev
```

Comment:
You probably want to add `libtool-bin` here.

Reply:
Interesting, my xenial vm image wanted libtool and not libtool-bin.

Reply:
/usr/bin/libtool on xenial comes from the [libtool-bin](https://packages.ubuntu.com/xenial/amd64/libtool-bin/filelist) package; the [libtool](https://packages.ubuntu.com/xenial/all/libtool/filelist) package only has /usr/bin/libtoolize.

(I don't know why.  It looks like trusty did not even have a libtool-bin package.)